actions/add_org.py

```diff
@@ -0,0 +1,26 @@
+from lib.base import BaseGithubAction
+import json
+from st2client.client import Client
+from st2client.models import KeyValuePair
+
+__all__ = [
+    'AddOrgAction'
+]
+
+
+class AddOrgAction(BaseGithubAction):
+    def run(self, user, url, token, github_type):
+
+        client = Client()
+        gitorgs = client.keys.get_by_name(name='git-orgs', decrypt=True)
+        if gitorgs:
+            dict=json.loads(gitorgs.value)
+        else:
+            dict={}
+        org = {'urer': user, 'url': url, 'token': token, 'type': github_type}
+        dict[user+'@'+url]=org
```

Comment:
I want to avoid the use of the character in the url as the separate character.  The key must be just the combination of the url and the user.  I'll use '|' instead of '@'.

Reply:
Ok, but can it be `url|blah`?

Reply:
Yes, the key doesn't need to make any sense.  It just a combined value of the user and the url.